src/controller/AutoCommissioner.h

```diff
@@ -23,6 +23,9 @@
 namespace chip {
 namespace Controller {
 
+constexpr size_t kAttestationElementsLength  = 900;
```

Comment:
Can you please repace this one with:
```
// As per specifications section 11.22.5.1. Constant RESP_MAX
constexpr size_t kMaxRspLen = 900;
```
^^^ src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

We should move this constant in `src/credentials/DeviceAttestationConstructor.h` most likely, and it would allow reuse here and in op creds cluster and in src/credentials/attestation_verifier/DefaultDeviceAttestationVerifier.cpp/DacOnlyPartialAttestationVerifier.cpp

Reply:
addressed